sdk-lib/src/main/java/cash/z/ecc/android/sdk/SdkSynchronizer.kt

```diff
@@ -417,6 +425,30 @@ class SdkSynchronizer private constructor(
         processor.refreshWalletSummary()
     }
 
+    override suspend fun refreshExchangeRateUsd() {
```

Comment:
We should update the Javadoc in the parent `Synchronizer` to reflect the fact that this function may produce exceptions. Ideally, we throw a custom exception, as commented above.